test/openshift/e2e/operator_expectations.go

```diff
@@ -26,7 +26,51 @@ const (
 	waitLong   = 10 * time.Minute
 )
 
-func (tc *testConfig) ExpectOperatorAvailable() error {
+func newWorkLoad() *batchv1.Job {
+	backoffLimit := int32(4)
```

Comment:
could also `k8s.io/utils/pointer` and `pointer.Int64Ptr`

Reply:
This code was just moved, so would be a follow-up PR.